src/plugins/themes.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const colors = require("colors/safe");
+const path = require("path");
+const themes = new Map();
+
+module.exports = {
+	getAll: getAll,
+	filename: filename
+};
+
+fs.readdir("client/themes/", (err, builtInThemes) => {
+	if (err) {
+		return;
+	}
+	builtInThemes
+		.filter((theme) => theme.endsWith(".css"))
+		.map(makeLocalThemeObject)
+		.forEach((theme) => themes.set(theme.name, theme));
+});
+
+fs.readdir(Helper.getPackagesPath(), (err, packages) => {
+	if (err) {
+		return;
+	}
+	packages
+		.map(makePackageThemeObject)
+		.forEach((theme) => {
+			if (theme) {
+				themes.set(theme.name, theme);
+			}
+		});
+});
+
+function getAll() {
+	return Array.from(themes.values());
```

Comment:
Could you do something like this:

```diff
───────────────────────────────
modified: src/plugins/themes.js
───────────────────────────────
@ themes.js:7 @ const fs = require("fs");
 const Helper = require("../helper");
 const colors = require("colors/safe");
 const path = require("path");
+const _ = require("lodash");
 const themes = new Map();

 module.exports = {
@ themes.js:39 @ fs.readdir(Helper.getPackagesPath(), (err, packages) => {
 });

 function getAll() {
-   return Array.from(themes.values());
+   return _.sortBy(Array.from(themes.values()), "displayName");
 }
```

This sorts the themes in the list, for example:

<img width="537" alt="screen shot 2017-09-17 at 01 27 27" src="https://user-images.githubusercontent.com/113730/30518302-0bc09cd4-9b48-11e7-8da1-a7e2e85b59bd.png">